src/Groups/GAPGroups.jl

```diff
@@ -508,7 +517,13 @@ end
 Base.:^(x::T, y::T) where T <: GAPGroupElem = group_element(_maxgroup(parent(x), parent(y)), x.X ^ y.X)
 
 """
-    isconjugate(G::Group, x::GAPGroupElem, y::GAPGroupElem)
+    isconjugate(G::GAPGroup, x::GAPGroupElem, y::GAPGroupElem)
+Return whether `x` and `y` are conjugate elements in `G`.
```

Comment:
These empty separator lines are important! Please, please add them everywhere, there are still a bunch of docstrings in this PR which are missing them.
```suggestion

Return whether `x` and `y` are conjugate elements in `G`.
```